src/injector/inject.ts

```diff
@@ -178,67 +253,7 @@ export function inject (vm: ChibiCompatibleVM) {
 
     // Blockly stuffs
     setTimeout(() => {
-        const blockly = window.chibi.blockly = getBlocklyInstance();
```

Comment:
the new ways to get Blockly look exciting. but I still want to keep this alternative method to maintain compatibility as much as possible.

Reply:
Okay.